tools/SourceKit/CMakeLists.txt

```diff
@@ -239,9 +239,9 @@ macro(add_sourcekit_executable name)
         PROPERTIES
```

Comment:
Could you change this invocation to be `set_property(TARGET "..." APPEND_STRING PROPERTY LINK_FLAGS)`?  Then you won't need to duplicate the code coverage flags below?